### PR TITLE
Cli parameter for GUI client + QT_LOGGING_RULES

### PR DESCRIPTION
**Pull request by michaelstingl**
@EParzefall @mmattel needs some linking, formatting, polishing etc

### Testing the PR

```
git clone git@github.com:owncloud/docs-client-desktop.git
cd docs-client-desktop
git advanced/cli-parameter
yarn install
yarn antora
yarn serve
```

Then you can find the changed version here:
- http://localhost:8080/desktop/next/advanced_usage/command_line_parameters.html
- http://localhost:8080/desktop/next/advanced_usage/environment_variables.html

### CODE REVIEW

File: modules/ROOT/pages/advanced_usage/environment_variables.adoc [hunk 1]
```diff
@@ -154,4 +154,8 @@ Set to -1 to never require full local discovery.
 | `OCC_UPDATE_URL`
 | unset
 | This value can be set in the environment of the client to let the client choose a different update URL for testing purposes. 
+
+| `QT_LOGGING_RULES`
+| unset
+| This value can be used to control the log content, You can find examples here: troubleshooting.html#control-log-content and here: https://community.kde.org/Guidelines_and_HOWTOs/Debugging/Using_Error_Messages#Controlling_Messages
```
Comment (mmattel) [inline modules/ROOT/pages/advanced_usage/environment_variables.adoc:160]:
```suggestion
| This value can be used to control the log content, see examples at xref:appendices/troubleshooting.adoc#control-log-content[Control Log Content] and  https://community.kde.org/Guidelines_and_HOWTOs/Debugging/Using_Error_Messages#Controlling_Messages[Controlling Messages].
```

**Comment by mmattel** [comment]
*  I did a build test and found that an envvar was double present - fixed.
* @michaelstingl to which releases does this need backporting?
3.2 and 3.1 ?

**Comment by michaelstingl** [comment]
All 3.x okay

**Review by mmattel — APPROVED** [review]

File: modules/ROOT/pages/advanced_usage/command_line_parameters.adoc [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+{description}
+
+== Command Description
+
+[source,console]
+----
+owncloud [options]
+----
+
+Command line paramters supported by `owncloud` include the following:
```
Comment (phil-davis) [inline modules/ROOT/pages/advanced_usage/command_line_parameters.adoc:17]: Typo pf "paramters" has been fixed in PR #399

**Pull request merged**